### ISSUE TITLE
Tooltips cannot be dismissed without moving focus off the target element

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1215]
Tooltips cannot be dismissed without moving the focus off of the target element (using ESC key for tooltips on a dialog results in the entire dialog being closed). As per usability guidelines, tooltips created using anything other than ```<a title="">``` need to be dismissable without moving focus from the target element.

**Comment by rmmayo** [comment]
PR: #1450

**Comment by mosefer** [comment]
Awesome! One minor exception - the help icons in the Settings -> Email -> Email Template Settings cannot be tab navigated

**Comment by rmmayo** [comment]
spoke with @mosefer - since the email header/footer tabs use the `inline-help` component which also uses the help message as the `aria-label` attribute, the screen reader already speaks the help message. tab navigation would cause the screen reader to read the message twice.
